tests/test_meshtastic_utils_connect_paths.py

```
@@ -357,6 +359,420 @@ def test_connect_meshtastic_logs_firmware_version_on_success(
+    assert result is mock_client
+    assert saw_reset_state_during_get_node_info is True
+    assert mu._relay_rx_time_clock_skew_secs == 5_000.0
+    assert mu._relay_startup_drain_deadline_monotonic_secs is None
+    assert mu._relay_reconnect_prestart_bootstrap_deadline_monotonic_secs is None
+    assert not mu._health_probe_request_deadlines
+    mock_subscribe.assert_not_called()
+    mock_submit_coro.assert_not_called()
+
+
+def test_connect_meshtastic_schedules_one_shot_probe_when_periodic_health_disabled(
+    reset_meshtastic_globals,
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)